### DISCUSSION TITLE
Multiple Recaptcha Variables installed after EWZ bundle installation

### DISCUSSION THREAD

**Discussion by nlipkanov** [sulu/SuluFormBundle#338] [Q&A] [answered]
Is there a reason why there are following recaptcha keys after installing the EWZ recaptcha bundle?:

###> google/recaptcha ###
GOOGLE_RECAPTCHA_SITE_KEY=
GOOGLE_RECAPTCHA_SECRET=
###< google/recaptcha ###

###> excelwebzone/recaptcha-bundle ###
EWZ_RECAPTCHA_SITE_KEY=
EWZ_RECAPTCHA_SECRET=
###< excelwebzone/recaptcha-bundle ###

Which keys do I use? The EWZ? The google ones?

**Comment by alexander-schranz** [comment, marked as answer]
@nlipkanov we are only supporting the EWZ Bundle as documented. The variables are not created by SuluFormBundle. That behaviour is part of symfony/flex and the flex recipes: https://github.com/symfony/recipes-contrib

**Reply by nlipkanov** [reply]
Yeah, I think my question is also actually at the wrong repo.
